src/main/java/uk/gov/dvsa/model/cvs/IVA30.java

```diff
@@ -4,12 +4,16 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import uk.gov.dvsa.model.Document;
 import uk.gov.dvsa.model.cvs.certificateData.IvaFailCertificateData;
+import uk.gov.dvsa.model.cvs.certificateData.Signature;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class IVA30 extends Document {
     @JsonProperty("ID")
     private String id;
 
+    @JsonProperty("Signature")
```

Comment:
Is it worth adding a verifySignature test to the IVA30Tests so we cover this new element.

Reply:
If it is going to be used in the near future then yes, I think that this would be worthwhile doing now although the template will need updating at that later date.